demos/smart_classroom_demo_gapi/main.cpp

```diff
@@ -628,90 +1002,178 @@ int main(int argc, char* argv[]) {
                 ie.SetConfig({{PluginConfigParams::KEY_DYN_BATCH_ENABLED, PluginConfigParams::YES}}, "GPU");
             }
 
-            if (FLAGS_pc)
-                ie.SetConfig({{PluginConfigParams::KEY_PERF_COUNT, PluginConfigParams::YES}});
-
             loadedDevices.insert(device);
         }
 
-        std::unique_ptr<AsyncDetection<DetectedAction>> action_detector;
+        // NOTE: Empty strings instead empty object of Params class 
+        cv::gapi::ie::Params<custom::FaceDetector> det_net("","","");
+        cv::gapi::ie::Params<custom::LandmarksDetector> landm_net("","","");
+        cv::gapi::ie::Params<custom::FaceReidentificator> reident_net("","","");
+        cv::gapi::ie::Params<custom::PersonDetActionRec> action_net("", "", "");
```

Comment:
Can those be declared without this `("","","")` thing? Does it make sense here to declare it like this?

Reply:
`cv::gapi::ie::Params` hasn't default constructor.